### PR TITLE
Adds Wiki Command 2: Github please work Boogaloo

### PR DESCRIPTION
**Pull request by AutisticFroggy**
Literally copy pasted off of mentor code

### CODE REVIEW

File: models/Discord/Commands/DiscordCommandWiki.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "https://i.imgur.com/RYM4zIO.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734845830900744202/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734846144701792256/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734844866583986256/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734844514069643294/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734844202998956076/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734844239200124938/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734844114482495518/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734844019624116274/unknown.png",
+        "https://cdn.discordapp.com/attachments/607402553143197696/734843632812818472/unknown.png",
```
Comment (swissloaf) [inline models/Discord/Commands/DiscordCommandWiki.js:40]: needs no comma on last line

Comment (AutisticFroggy) [reply models/Discord/Commands/DiscordCommandWiki.js:40]: it's JS it's fine

Comment (AutisticFroggy) [reply models/Discord/Commands/DiscordCommandWiki.js:40]: JS lists autoresolves these things anyway

Comment (JamieD1) [inline models/Discord/Commands/DiscordCommandWiki.js:40]:
```suggestion
        "https://cdn.discordapp.com/attachments/607402553143197696/734843632812818472/unknown.png"
```

**Pull request merged**